src/ext/java/org/opentripplanner/ext/vehiclerentalservicedirectory/api/NetworkParameters.java

```diff
@@ -5,15 +5,19 @@
 /**
  * Parameters for a specific network.
  * <p>
- * The {@link GbfsVehicleRentalDataSourceParameters} support  {@code overloadingAllowed} and
- * {@code allowKeepingRentedVehicleAtDestination} is not included here since they are not part of
- * the GBFS specification. If there is a demand for these, they can be added.
+ * The {@link GbfsVehicleRentalDataSourceParameters} supports {@code overloadingAllowed}
+ * which is not included here since they are not part of
+ * the GBFS specification. If there is a demand for it, it can be added.
  * <p>
  * @param network The network name
  * @param geofencingZones enable geofencingZones for the given network
  */
```

Comment:
Maybe add param documentation here since the scope of allowKeepingAtDestination is not the most clear. However, idk if it makes sense to duplicate it's explanation in multiple places.